### DISCUSSION TITLE
Switch to flatpak based distro

### DISCUSSION THREAD

**Discussion by denisok** [openSUSE/openSUSEway#130] [Polls]
currently, there are some applications installed as packages, such as firefox.

Idea is to switch to the flatpak completely, and pre-install such packages from the flatpak. It is done in this way in the https://github.com/ProjectGreybeard/branding/tree/main/config/firstboot project

**Comment by FilippoBonazziSUSE** [comment]
I don't really want to switch to flatpak on my normal TW desktop. And in openSUSEway we really only need recommended applications (such as Firefox) to be installed - we don't care where they come from. So users should be able to provide applications from wherever, and the openSUSEway config should still work
